samples/openapi3/client/petstore/typescript/builds/default/types/ObjectParamAPI.ts

```diff
@@ -142,7 +142,7 @@ export class ObjectPetApi {
      * Finds Pets by status
      * @param param the request object
      */
-    public findPetsByStatus(param: PetApiFindPetsByStatusRequest, options?: Configuration): Promise<Array<Pet>> {
+    public findPetsByStatus(param: PetApiFindPetsByStatusRequest , options?: Configuration): Promise<Array<Pet>> {
```

Comment:
Can we remove that one additional space in the mustache? :)

Reply:
removed :)